pkg/asset/ignition/bootstrap/common.go

```diff
@@ -615,6 +616,15 @@ func (a *Common) addParentFiles(dependencies asset.Parents) {
 
 		// Replace files that already exist in the slice with ones added later, otherwise append them
 		for _, file := range ignition.FilesFromAsset(rootDir, "root", 0644, asset) {
+			// We limit read access to the fencing secrets
+			match, err := regexp.MatchString(fmt.Sprintf("%s%s", ".*", fmt.Sprintf(machines.FencingCredentialsSecretFileName, "[0-9]+")), file.Path)
+			if err != nil {
+				logrus.Error("failed regex scan for fencing secrets during ignition files creation")
```

Comment:
```suggestion
				logrus.Warnf("failed regex scan for fencing secrets during ignition files creation: %s", err.Error())
```

nit: error message might be helpful :D Also, should we use only `WARN` here since the install proceeds anyway?